src/Conf.ml

```diff
@@ -439,6 +440,15 @@ module Formatting = struct
     C.choice ~names ~all ~doc ~section (fun conf x ->
         {conf with break_infix= x} )
 
+  let break_infix_bind =
+    let doc =
+      "Force infix bind expressions to break before the operator if it \
+       does not fit on a single line."
```

Comment:
I'm a little uncomfortable with the name and description of this option, which only mentions 'bind', while the implementation applies to any infix op with a fun(ction) expression as second argument. Either specialize the implementation or generalize the description.